README.md

```diff
@@ -1,3 +1,67 @@
 schema
 ======
-[![Build Status](https://travis-ci.org/gorilla/schema.png?branch=master)](https://travis-ci.org/gorilla/schema)
+[![GoDoc](https://godoc.org/github.com/gorilla/schema?status.svg)](https://godoc.org/github.com/gorilla/schema) [![Build Status](https://travis-ci.org/gorilla/schema.png?branch=master)](https://travis-ci.org/gorilla/schema)
+
+Package gorilla/schema fills a struct with form values.
+
+## Example
+
+Here's a quick example: we parse POST form values and then decode them into a struct:
+
+```go
+type Person struct {
+    Name  string
+    Phone string
+}
+
+func MyHandler(w http.ResponseWriter, r *http.Request) {
+    err := r.ParseForm()
+
+    if err != nil {
+        // Handle error
+    }
+
+    decoder := schema.NewDecoder()
+    // r.PostForm is a map of our POST form values
+    err := decoder.Decode(person, r.PostForm)
+
+    if err != nil {
+        // Handle error
+    }
+
+    // Do something with person.Name or person.Phone
+}
+```
+Note: it is a good idea to set a Decoder instance as a package global, because it caches meta-data about structs, and a instance can be shared safely:
+
+```go
+var decoder = schema.NewDecoder()
```

Comment:
how about just make this a global in the above example, and the text after "Note:" as a comment. Might as well illustrate the most common usage.